django-stubs/db/models/options.pyi

```diff
@@ -122,6 +122,14 @@ class Options(Generic[_M]):
     def get_fields(
         self, include_parents: bool = ..., include_hidden: bool = ...
     ) -> list[Field[Any, Any] | ForeignObjectRel | GenericForeignKey]: ...
+    def _get_fields(
+        self,
+        forward: bool = ...,
+        reverse: bool = ...,
```

Comment:
We could probably have an overload on `reverse`, I think if `reverse=False`, `ForeignObjectRel` can't be present in the return type